keras_cv/models/csp_darknet.py

```diff
@@ -177,7 +180,230 @@ def CSPDarkNet(
+    pooling=None,
+    classifier_activation="softmax",
+    name="CSPDarkNetTiny",
+    **kwargs,
+):
+    return CSPDarkNet(
+        depth_multiplier=DEPTH_MULTIPLIERS["tiny"],
+        width_multiplier=WIDTH_MULTIPLIERS["tiny"],
+        include_rescaling=include_rescaling,
+        include_top=include_top,
+        use_depthwise=use_depthwise,
+        classes=classes,
+        weights=weights,
+        input_shape=input_shape,
+        input_tensor=input_tensor,
+        pooling=pooling,
+        classifier_activation=classifier_activation,
+        name=name,
+        **kwargs,
+    )
+
+
+def CSPDarkNetS(
+    include_rescaling,
+    include_top,
+    use_depthwise=False,
+    classes=None,
+    weights=None,
+    input_shape=(None, None, 3),
+    input_tensor=None,
+    pooling=None,
+    classifier_activation="softmax",
+    name="CSPDarkNetS",
+    **kwargs,
+):
+    return CSPDarkNet(
+        depth_multiplier=DEPTH_MULTIPLIERS["s"],
+        width_multiplier=WIDTH_MULTIPLIERS["s"],
+        include_rescaling=include_rescaling,
+        include_top=include_top,
+        use_depthwise=use_depthwise,
+        classes=classes,
+        weights=weights,
+        input_shape=input_shape,
+        input_tensor=input_tensor,
+        pooling=pooling,
+        classifier_activation=classifier_activation,
+        name=name,
+        **kwargs,
+    )
+
+
+def CSPDarkNetM(
+    include_rescaling,
+    include_top,
+    use_depthwise=False,
+    classes=None,
+    weights=None,
+    input_shape=(None, None, 3),
+    input_tensor=None,
+    pooling=None,
+    classifier_activation="softmax",
+    name="CSPDarkNetM",
+    **kwargs,
+):
+    return CSPDarkNet(
+        depth_multiplier=DEPTH_MULTIPLIERS["m"],
+        width_multiplier=WIDTH_MULTIPLIERS["m"],
+        include_rescaling=include_rescaling,
+        include_top=include_top,
+        use_depthwise=use_depthwise,
+        classes=classes,
+        weights=weights,
+        input_shape=input_shape,
+        input_tensor=input_tensor,
+        pooling=pooling,
+        classifier_activation=classifier_activation,
+        name=name,
+        **kwargs,
+    )
+
+
+def CSPDarkNetL(
+    include_rescaling,
+    include_top,
+    use_depthwise=False,
+    classes=None,
+    weights=None,
+    input_shape=(None, None, 3),
+    input_tensor=None,
+    pooling=None,
+    classifier_activation="softmax",
+    name="CSPDarkNetL",
+    **kwargs,
+):
+    return CSPDarkNet(
+        depth_multiplier=DEPTH_MULTIPLIERS["l"],
+        width_multiplier=WIDTH_MULTIPLIERS["l"],
+        include_rescaling=include_rescaling,
+        include_top=include_top,
+        use_depthwise=use_depthwise,
+        classes=classes,
+        weights=parse_weights(weights, include_top, "cspdarknet"),
+        input_shape=input_shape,
+        input_tensor=input_tensor,
+        pooling=pooling,
+        classifier_activation=classifier_activation,
+        name=name,
+        **kwargs,
+    )
+
+
+def CSPDarkNetX(
```

Comment:
Should this be XL instead of X, or is X a convention for CSP Darket

Reply:
The YoloX model corresponding to this is called YoloX-x. Hence, the name.